src/net.h

```diff
@@ -351,6 +354,7 @@ class CConnman
     unsigned int nSendBufferMaxSize;
     unsigned int nReceiveFloodSize;
 
+    bool m_relay_txes;
```

Comment:
Some future refactor can call `GetRelayTxes` (or other) before `Start()` and there is no way to know/prevent that.